### PR TITLE
Feature/card goal

### PR DESCRIPTION
**Pull request by Mickaellah**
Hi Shaun,

This is the component that we asked to build. Please have a look at it.
Problems that we were facing during this task: 

  - copying three paths, which is the computer icon into one SVG file.
  - struggled for making sure the SVGs icons.
  - setting border radius for the icons' background.

### CODE REVIEW

File: css/card-goal.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+.card_goal_navigation__image {
+  max-width: 80px;
+  max-height: 80px;
+  background-color: #00B2D8;
+  transform: translateY(-70%);
+  align-self: center;
+  align-items: flex-start;
+  border-radius: 50%;
+}
+
```
Comment (Tsikitsiky) [inline css/card-goal.css]: What css did you update?

File: css/card-goal.css [hunk 2]
```diff
@@ -41,10 +41,10 @@
...
-  padding-inline-end: 20px;
-  padding-block-end: 83px;
-  padding-block-start: 23px;
+  padding-left: 20px;
+  padding-right: 20px;
+  padding-bottom: 83px;
+  padding-top: 23px;
   margin: 0;
   font-size: 14px;
   line-height: 22px;
```
Comment (Ny-Rianala) [inline css/card-goal.css]: It looks nice. However,  the commit message doesn't describe what you really did.

File: css/card-goal.css [hunk 3]
```diff
@@ -60,4 +58,18 @@
...
+    -ms-grid-template-columns: repeat(2, 1fr);
+  }
+}
 
+@media(min-width: 1000px) {
+  .card_goal_navigation_mobile__list {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    -ms-grid-template-columns: repeat(3, 1fr);
+  }
```
Comment (Ny-Rianala) [inline css/card-goal.css]: In general, it looks good. It is nice seeing you using the new property '-ms-grid-template-columns'. The commit message should have been more specific.

File: css/card-goal.css [hunk 4]
```diff
@@ -1,3 +1,7 @@
+header {
```
Comment (Tsikitsiky) [inline css/card-goal.css]: Should be more specific with the header selector, what if there is another header somewhere? But in all it looks nice

**Comment by Ny-Rianala** [comment]
-There is nothing which is fixed in this file `fixing css`.
-Please, specify what changes you made in the file.
-Commit messages should be more specific.

**Review by Ny-Rianala — COMMENTED** [review]
Hello team,

Overall, code is good but there are things should be improved.

Thanks,
Kati and Sarah

File: css/card-goal.css (hunk 3 above)
Comment (Mickaellah) [reply css/card-goal.css]: Thank you team for this review. We will improve our code.

File: css/card-goal.css (hunk 2 above)
Comment (Mickaellah) [reply css/card-goal.css]: Thank you! We will write good commit messages for what we have done.

File: css/card-goal.css (hunk 4 above)
Comment (Mickaellah) [reply css/card-goal.css]: Thank you so much! We have made a change for this selector.

File: css/card-goal.css (hunk 1 above)
Comment (Mickaellah) [reply css/card-goal.css]: Thanks for this suggestion. We will write specific commit messages next time.

**Pull request closed without merging**